tensorflow/java/src/test/java/org/tensorflow/TensorTest.java

```diff
@@ -105,55 +116,90 @@ public void createFromBufferWithNonNativeByteOrder() {
   }
 
   @Test
-  public void createWithTypedBuffer() {
-    int[] ints = {1, 2, 3, 4};
-    float[] floats = {1f, 2f, 3f, 4f};
-    double[] doubles = {1d, 2d, 3d, 4d};
-    long[] longs = {1L, 2L, 3L, 4L};
-    long[] shape = {4};
+  public void whenCreateUsingDoubleBufferThenExpectedMatrixIsStoredInTensor() {
+    final double[] expected = {1d, 2d, 3d, 4d};
+    final long[] shape = {4};
+
+    try (final Tensor t = Tensor.create(shape, DoubleBuffer.wrap(expected))) {
+      final double[] actual = new double[expected.length];
+      t.copyTo(actual);
+      assertArrayEquals(expected, actual, EPSILON);
```

Comment:
Readability is in the eye of the reader :). I find the single line more readable, but I don't feel too strongly and appreciate that others may legitimately differ, so you're welcome to keep the three lines if you wish.